purchases/src/main/kotlin/com/revenuecat/purchases/identity/IdentityManager.kt

```diff
@@ -135,11 +147,17 @@ internal class IdentityManager(
         }
     }
 
-    private fun invalidateCustomerInfoAndETagCacheIfNeeded(appUserID: String) {
+    private fun invalidateCustomerInfoAndETagCacheIfNeeded(
+        appUserID: String,
+        cacheEditor: SharedPreferences.Editor,
+    ) {
+        if (backend.verificationMode == SignatureVerificationMode.Disabled) {
```

Comment:
I guess this is to avoid the `getCachedCustomerInfo` below from happening when it's disabled right? Makes sense!

Reply:
exactly! I noticed we were getting the customer info all the time! I will add a comment here to make sure it's clear